docs/self-managed/platform-deployment/docker.md

```diff
@@ -35,6 +35,16 @@ This will give you a single broker node with the following ports exposed:
 - `26501`: Command API (internal, gateway-to-broker)
 - `26502`: Internal API (internal, broker-to-broker)
 
+### Multi-platform support
+
+With the Camunda Platform 8.2.0 release and onward, all Camunda Platform 8 Docker images are provided as multi-platform images natively supporting the following platforms:
```

Comment:
What do you mean by "natively supporting?" I haven't heard this before so just curious.

Reply:
The background for this is, that if a docker image is based on a different architecture than the host, an emulation layer needs to run, such as [binfmt](https://github.com/tonistiigi/binfmt).

Tooling like Docker Desktop might abstract that away from the user, who might only notice that this is happening at all due to poor performance of the emulation layer. We saw that the amd64 emulation by docker on Apple M1 Macs yielded "unable to use" performance ^^.

In case a native image in relation to the docker host is available though, the docker client will already use that automatically when pulling from the registry.

Reply:
Thank you for these details!